src/components/Scaffolding/Scaffolding.jsx

```diff
@@ -22,31 +22,54 @@ import {
     return input.replace(/[^a-z0-9\s]/gi, '').replace(/(gt)/gi, '')
   }
   const Hit = ({ hit }) => {
-    console.log(hit);
   
     return (
       <article className="hit-card">
-        <a target="_blank" rel="noopener noreferrer"
-          href={hit.repository && hit.repository.url ? hit.repository.url: '/'}>
-          <div className="product-wrapper">
-              <div className="product-summary">
-                <div className="product-title">
-                <h3>{hit.name}</h3> 
-                </div>
-                <div className="product-avatar">
-                  <img src={hit.owner.avatar} height="40" width="40"/>
-                </div>
+       {
+          hit.repository && hit.repository.url ? (
+            <a target="_blank" rel="noopener noreferrer"
+              href={hit.repository && hit.repository.url ? hit.repository.url: '/'}>
```

Comment:
No need to check twice? :)

Reply:
Try to find the npm link if gh is not present. Agnolia grabs data from npm registry

Reply:
Fixed the approach, first check is needed if hit is available so we can render because if hit's not available list won't be rendered, 2nd is to check is gh link is present, if not use npm link.